### DISCUSSION TITLE
[Fixed] Brave browser - Redirect to app fails after confirm login permission via browser

### DISCUSSION THREAD

**Discussion by K-eL** [community/community#64844] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Unable to open the app because it does not redirect to the app after giving the permission via browser.

Steps to reproduce:
- install the app
- login with first option (opens browser)
- login on github page
- confirm the permission on github to the app (expected to be redirected to the app) - nothing happens
- click on redirect link - nothing happens
- refresh the page (browser show the page wants to open the app)
- confirm - nothing happens.

Samsung S20+; SM-G985F/DS
Android 13; Build/TP1A.220624.014
Brave 1.57.50, Cromium 116.0.5845.96

**Comment by aconsuegra** [comment]
Hi @K-eL, thanks for reaching out and sorry about the inconveniences. Could you share which version of the Android app are you using?

We recently shipped some sign in improvements to our Beta build, so I just want to make sure that we didn't ship a regression if you are using the Beta build. In case you aren't, would you mind joining the Beta testers program and test if the behavior improves for you?

**Reply by K-eL** [reply]
Hi @aconsuegra , same problem is happening on Beta version.

**Reply by aconsuegra** [reply, marked as answer]
Thanks for testing!

Could you double check your Brave settings and make sure that "Allow app links to open outside of Brave" is enabled?

![Screenshot_20230824-174719_Brave](https://github.com/community/community/assets/1814882/32fe2f1e-40a1-40b8-8361-9907994d3923)

**Reply by K-eL** [reply]
Yes, that solved the problem. Which is weird, because when the browser asks to allow the page to open the app, and I confirm, the app still didn't open. Now works after changing this option. Thanks!

**Reply by Achram35** [reply]
Ho

**Reply by developer982** [reply]
🙄

**Reply by lnsantos** [reply]
🙄

**Reply by Saud9911** [reply]
😑

**Reply by sadall07** [reply]
> Thanks for testing!
> 
> Could you double check your Brave settings and make sure that "Allow app links to open outside of Brave" is enabled?
> 
> ![Screenshot_20230824-174719_Brave](https://private-user-images.githubusercontent.com/1814882/263045823-32fe2f1e-40a1-40b8-8361-9907994d3923.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQyNTgzNDYsIm5iZiI6MTcyNDI1ODA0NiwicGF0aCI6Ii8xODE0ODgyLzI2MzA0NTgyMy0zMmZlMmYxZS00MGExLTQwYjgtODM2MS05OTA3OTk0ZDM5MjMuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MjFUMTYzNDA2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YmE5YmQ3MDMxOGQ4MTYyYjVmYzcxODQ2OGJmZDU1OTg2NjRlMDk0OTRhYWEwZGY0YmY4ZTNkY2M1ZmZhZDYzZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.7gFBUoA865tO_7GK1XDr5J0Zxekl1GkiKzpwxHxprMc)

**Reply by Shawnn9** [reply]
🙄

**Reply by Javiertiolico** [reply]
[`/>`](url) Thanks for testing!
> 
> Could you double check your Brave settings and make sure that "Allow app links to open outside of Brave" is enabled?
> 
> ![Screenshot_20230824-174719_Brave](https://private-user-images.githubusercontent.com/1814882/263045823-32fe2f1e-40a1-40b8-8361-9907994d3923.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5MDY0NTUsIm5iZiI6MTczMzkwNjE1NSwicGF0aCI6Ii8xODE0ODgyLzI2MzA0NTgyMy0zMmZlMmYxZS00MGExLTQwYjgtODM2MS05OTA3OTk0ZDM5MjMuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTFUMDgzNTU1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzE3YjkxODE5ZWExNTNiNThkYTU1M2JmMmM1MDEyMTc2MWRkNjk3ZGU4NTVhMzRjZGY1NDQ2NzYwOGM0NjEyZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.xLcMxI7Kj2l94e5MIdG7WcvDLcMEae0_Hh4-m7kINM4)

**Comment by romashka3073** [comment]
cool

**Comment by shahinkhorshidfar** [comment]
[]()

**Comment by MrNiSso** [comment]
🔴Can u ratings my app and give me to /10🌟

https://play.google.com/store/apps/details?id=com.fd.food_recipe0

**Comment by joel8485** [comment]
9

**Comment by 07091992045** [comment]
OTP

**Comment by MERTGANI** [comment]
[](url)

**Comment by JorenVsyntra** [comment]
Hey everyone!

**Comment by Shawnn9** [comment]
cool
